addOns/soap/soap.gradle.kts

```diff
@@ -17,7 +17,8 @@ zapAddOn {
 
 dependencies {
     implementation("com.predic8:soa-model-core:1.6.0")
-    implementation("com.sun.xml.ws:jaxws-rt:2.3.2")
+    implementation("jakarta.xml.soap:jakarta.xml.soap-api:1.4.2")
+    implementation("com.sun.xml.messaging.saaj:saaj-impl:1.5.2")
```

Comment:
Are either of these actually used? I don't see any imports or occurrences of "saaj" or "soap-api".

Reply:
Yup, the `java.xml.soap.*` imports require the `jakarta.xml.soap-api` package because these modules were removed with the release of Java 11 (see https://stackoverflow.com/a/54574003/5511659). The `saaj-impl` package is required for similar reasons (see https://stackoverflow.com/a/52349146/5511659).